citus-enterprise.spec

```diff
@@ -3,6 +3,7 @@
 %global pginstdir /usr/pgsql-%{pgpackageversion}
 %global sname citus-enterprise
 %global pname citus
+%global debug_package %{nil}
```

Comment:
It turns out that the debug symbols should actually be in the main package now. The reason that I had to add this line was because `rpmbuild` could not exctract the debug symbols anymore, since the .so file is now encrypted. This means that they are also not stripped out of the `.so` file, so they should be in the `.so` file after decrypting.